apps/designer/app/designer/features/topbar/publish.tsx

```diff
@@ -15,15 +15,20 @@ import {
   useId,
 } from "@webstudio-is/design-system";
 import { useIsPublishDialogOpen } from "../../shared/nano-states";
+import env from "~/shared/env";
 
 type PublishButtonProps = { project: Project };
 
-const host =
-  typeof location === "object"
-    ? location.host.includes("webstudio.is")
-      ? "wstd.io"
-      : location.host
-    : "";
+const host = () => {
+  if (env.PUBLISHER_ENDPOINT && env.PUBLISHER_DOMAIN !== "") {
+    return env.PUBLISHER_DOMAIN;
+  }
+  if (typeof location === "object") {
+    if (location.host.includes("webstudio.is")) return "wstd.io";
```

Comment:
can we avoid hardcoding this check, can't it be based on env?

Reply:
It could. Should we rename the `PUBLISHER_DOMAIN` to `SITES_DOMAIN` so it can be used application-wide to identify the domain?

Reply:
Oh we definitely should have one and the same name if its about naming the same thing, I guess publisher domain makes sense as a name

The other webstudio.is could be the DESIGNER_DOMAIN

Reply:
Could you please check if this is what you thinking of? Thanks in advance.